### ISSUE TITLE
🔤 IAP manager homepage - Swap out old Manager's Package for new Document

### ISSUE THREAD

**Issue by MCommonda** [GCTC-NTGC/gc-digital-talent#9189]
## 🔤 The current downloadable manager's package that is live will need to be swapped out with an updated document, for both French and English sites.

Describe the update or addition to the copy that you are proposing.

## 🕵️ The current document found here: 
This is the file that IS live but should be removed from our platform
https://view.officeapps.live.com/op/view.aspx?src=https%3A%2F%2Ftalent.canada.ca%2Fdocuments%2FManager%2520package%2520-%2520ITAPIP.docx&wdOrigin=BROWSELINK
will be replaced with a newer document provided by the IAP team, for both English and French sites

On our site its at https://talent.canada.ca/documents/Manager%20package%20-%20ITAPIP.docx

## 🎨 Design File

link(s) to be provided as soon as they have passed accessibility requirements. @RM-1978 

### 🧑‍🎨 Designer

@substrae 

## 🙋‍♀️ Proposed Implementation

New documents for swap: 
English: 
[IAP Manager's package for download - English.pptx](https://github.com/GCTC-NTGC/gc-digital-talent/files/14183648/IAP.Manager.s.package.for.download.-.English.pptx)

French:
[NEW_ITAP Managers Package Resource_FR _VT.pptx](https://github.com/GCTC-NTGC/gc-digital-talent/files/14181602/NEW_ITAP.Managers.Package.Resource_FR._VT.pptx)

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the copy has been added or updated appropriately.

- [ ] The new powerpoint files are downloadable instead of the original docx file
- [ ] New document downloads should have same file name just different file extension

**Comment by MCommonda** [comment]
New documents for swap: 
English: 
[Uploading NEW_ITAP Managers Package Resource_EN _VT.pptx…]()

**Comment by tristan-orourke** [comment]
@MCommonda looks like the link to the English pptx file doesn't work, could you upload it again?

**Comment by MCommonda** [comment]
Just edited comment above! @tristan-orourke
